webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
+             double                        consentTotalRoundTripTime;
+             unsigned long long            consentRequestsReceived;
+             unsigned long long            consentRequestsSent;
+             unsigned long long            consentResponsesReceived;
+             unsigned long long            consentResponsesSent;
```

Comment:
Is this before or after consent is received?  Because I don't think it really matters how many consent requests are received and responses sent before we receive consent.  I think we could just remove those two because total requestsReceived and responsesSent is enough.